src/main/ruby/core/argf.rb

```diff
@@ -87,9 +88,11 @@ def binmode?
     #
     def close
       advance!
+      return if @stream.to_io == STDIN
```

Comment:
Should this return so early for STDIN or should it just skip the `@stream.close`?